src/main/java/core/basesyntax/NumberBall.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberBall {
+    public int getRandomNumber() {
+        Random randomNumber = new Random();
+        int value = randomNumber.nextInt(100);
```

Comment:
Magic number '100' should be extracted as a named constant to improve code readability and maintainability.

Reply:
add     private int maxNumberBall  = 100;